celobox/phantompasswd.py

```diff
@@ -76,7 +77,10 @@ def load_data(self, domain):
 
         for content_type in ContentTypes:
             try:
-                content = open(os.path.join(manifests_path, '{}.{}'.format(domain, content_type.value)))
+                content = open(resource_filename(
+                    __name__,
+                    os.path.join(manifests_path, '{}.{}'.format(domain, content_type.value))
+                ))
```

Comment:
This is a bit strange (though my fault for not flushing out the `todo`). At some point, I'd like to be able to supply a custom manifests path from the command line. So, perhaps the `resource_filename` call should move into [`_get_manifests_path`](https://github.com/scoates/celobox/blob/phantom/phantompasswd.py#L118).